client/app/components/message.coffee

```diff
@@ -22,7 +22,8 @@ LayoutActionCreator  = require '../actions/layout_action_creator'
 MessageActionCreator = require '../actions/message_action_creator'
 ContactActionCreator = require '../actions/contact_action_creator'
 
-RouterMixin           = require '../mixins/router_mixin'
+Router= require '../mixins/router_mixin'
```

Comment:
If it's not a mixin, it shouldnt be in mixins folder, may be router.coffee at client's root ?